di-ipv-core-stub/build.gradle

```diff
@@ -25,6 +25,12 @@ dependencies {
 			"com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.17.2"
 
 	testImplementation "org.junit.jupiter:junit-jupiter:5.10.3"
+
+	configurations.all {
+		resolutionStrategy {
+			force 'org.eclipse.jetty:jetty-server:9.4.55.v20240627' // https://github.com/govuk-one-login/ipv-stubs/security/dependabot/24
```

Comment:
Is it possible to set this as an explicit dependency, rather than overriding the resolution?

Reply:
It probably is, will take a look